<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveExitDialog</class>
 <widget class="QDialog" name="SaveExitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>93</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>93</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>93</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Возможно, какие-то изменения не были сохранены. Какие будут действия?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить и выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OnlyExitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти без сохранения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
